### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by SadiqZak**

### CODE REVIEW

**Review by MurtazaBagwala07 — APPROVED** [review]
looks good , just add some text to "alt" in avatar

File: CSS/components.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+.container{
+    width: 80%;
+    min-height: 100vh;
+    height: max-content;
+    margin: auto;   
+}
+
+.container-secondary{
+    width: 100%;
+    margin: 50px auto 0;
```
Comment (romabulani) [inline CSS/components.css]: Use rems instead of px

File: CSS/components.css [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    display: flex;
+    padding: 0 1rem;
+}
+
+.container-tertiary{
+    min-width: 0;
+    margin: 0px auto;
+    max-width: 800px;
```
Comment (romabulani) [inline CSS/components.css:39]: Here also use rem instead of px

**Review by romabulani — APPROVED** [review]
After these 2 changes of px to rem, good to merge

**Pull request merged**